lib/audio_pool.dart

```diff
@@ -11,11 +11,11 @@ typedef void Stoppable();
 /// All AudioPlayers loaded are for the same [sound]. If you want multiple sounds use multiple [AudioPool].
 /// Use this class if you'd like have extremely quick firing, repetitive and simultaneous sounds, like shooting a laser in a fast-paced spaceship game.
 class AudioPool {
-  AudioCache cache;
+  late AudioCache cache;
   Map<String, AudioPlayer> currentPlayers = {};
   List<AudioPlayer> availablePlayers = [];
 
-  String sound;
+  final String sound;
   bool repeating;
   int minPlayers, maxPlayers;
 
```

Comment:
unrelated bug but either one of these things should be done:
* make min, maxPlayers final and convert the constructor+init into a async factory
* make init clear the aP list so it can be called multiple times when people change min, max
right now it's in an inbetween state whether you can change it or not